.github/workflows/trivy.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         uses: aquasecurity/trivy-action@master
         with:
           # Path to Docker image
-          image-ref: "ghcr.io/catenax-ng/tx-vas-country-risk-frontend:main"
+          image-ref: "ghcr.io/catenax-ng/tx-vas-country-risk-frontend:latest"
```

Comment:
I think this should stay on `:main`. The default trivy scans should scan the default branch. 
@scherersebastian what do you think about that?

Reply:
done